wordpress-mobile/WordPress-iOS
# Issue/5467 image properties editing improvements

Fixes #5467 

To test:
 - Create a new post
 - Add an image to it
 - Tap on the image
 - Tap on the edit option
 - Changes options like title, caption, alt and link and check if they are updated correctly
 - Click done to see if they are updated on the post
 - Click on cancel to see if they are cancelled.

Needs review: @aerych
